CONTRIBUTING.md

```diff
@@ -307,7 +307,7 @@ notes about [commit squashing](#commit-squashing)).
 A good commit message should describe what changed and why.
 
 1. The first line should:
-   - contain a short description of the change (preferably 50 characters or less,
+   - contain a short description of the change (preferably 50 characters or fewer,
```

Comment:
Maybe my sense as a non-native speaker is off, but with `fewer`, re-ordering to `50 or fewer characters` sounds more natural to me? 😄

Reply:
It feels more correct to me too!

I live in the US. Interchanging `fewer` and `less` is common here, even when it's technically incorrect. It doesn't bother me much, actually. The distinction is pedantic.

However, I figured that I should fix it anyway. It would bother someone else eventually, I think.